x-pack/legacy/plugins/siem/public/pages/detection_engine/rules/components/rule_actions_field/index.tsx

```diff
@@ -0,0 +1,86 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React, { useCallback, useEffect, useState } from 'react';
+import deepMerge from 'deepmerge';
+
+// eslint-disable-next-line @kbn/eslint/no-restricted-paths
```

Comment:
Any context as to why it's okay to import these even though they're coming from a restricted path? Should they not be restricted?

Reply:
`loadActionTypes` are not exposed on top-level, but we need that to fetch current action types and then to filter only supported ones. It's a temporary solution, will talk to the alerting team to make sure `loadActionTypes` is exposed on the top-level
![image](https://user-images.githubusercontent.com/5188868/77460651-acb83f00-6e01-11ea-9b17-676397600a10.png)